### ISSUE TITLE
CTA up-to-date IRFs

### ISSUE THREAD

**Issue by micheledoro** [IndirectDarkMatterSearchesIFAE/ObservationOptimization#35]
IRFs of CTA changes with time as long as the instrument design and analysis evolve, are the scripts made in a way that they can make use of new IRFs produced by CTA?

**Comment by Jopana** [comment]
Yes, they are. Current version can read a 'predefined' CTA IRFs, or you can introduce your own IRFs by .root or .txt files. You can name these new IRFs as you prefer to name them.

**Comment by micheledoro** [comment]
It is good to write this down in the readme file.
